Sources/AblyAssetTrackingPublisher/WorkerQueue/PublisherWorkerQueueProperties.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+import AblyAssetTrackingInternal
+
+internal class PublisherWorkerQueueProperties: WorkerQueueProperties
```

Comment:
Can you please do a general check for `internal` in your PR and remove it? `internal` [is the default access level](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/accesscontrol/#Default-Access-Levels).

Reply:
ed9434f